woocommerce/woocommerce-ios
# 6264 Add Interac icon for IPP receipts

Closes: #6264


### Description

For In Person Payments in Canada, we will add Interac payments to the supported payment methods for 

### Testing instructions


Use commit 1cc4e57e19e58711eb5f994e687be1f2e9cedca6 to force all card receipts to be Interac branded. It is one commit on from this change, with an explicit `CardPresentTransactionDetails.init(from:)` method added which decodes the brand as `interact` when it's retrieved from storage.

1. In the Orders tab, tap + to add a simple payments order
2. Specify an amount, and tap through to card payment
3. Take payment, then tap `Continue to orders`
4. Tap the order you just added
5. Tap `See Receipt`
6. Observe that the preview of the receipt includes the Interac logo

N.B. if you open print or email receipt, the `CardPresentTransactionDetails.init` is not used, so the original card details are used. It's not currently possible to do a real Interac test transaction.

### Screenshots

![InteracReceipt@2x](https://user-images.githubusercontent.com/2472348/155337022-84eb6afe-afda-4e3d-8c39-e5d9873483cc.png)


---
